frontend/src/lib/api.ts

```diff
@@ -256,6 +265,28 @@ const api = {
         },
     },
 
+    activity: {
+        list(activityLogProps: ActivityLogProps, page: number = 1): Promise<CountedPaginatedResponse> {
+            const urlForScope: {
+                [key in ActivityScope]: (props: ActivityLogProps) => string
+            } = {
```

Comment:
nit: `Record<ActivityScope, (props: ActivityLogProps) => string>` (or maybe not a nit and this doesn't work for some reason)

Reply:
No, that seems to work 👍